src/utils/utils.ts

```diff
@@ -180,12 +180,8 @@ export const fetchJson = async (
   if (CLIENT_URL) {
     headers["Origin"] = CLIENT_URL.hostname;
   }
-  const token = await authService.getAccessToken();
-  if (token) {
-    // the Authorization header is already used for Basic auth,
-    // therefore we use the X-Authorization header for the OIDC token
-    headers["X-Authorization"] = "Bearer " + token;
-  }
+  if (additionalHeaders)
+    headers = Object.assign(headers, additionalHeaders)
```

Comment:
```suggestion
    Object.assign(headers, additionalHeaders)
```

This modifies the headers object in place, you don't need to re-assign it.